examples/live_language_change/browser_app_modules.R

```diff
@@ -0,0 +1,89 @@
+#' This script demonstrates how to use shiny.i18n Translator object
+#' for live language change on the UI side with Shiny modules. Two key steps are:
+#' (a) add `usei18n(i18n)` to UI
+#' (b) use `update_lang` function to change the language in session
+
+library(shiny)
+library(shiny.i18n)
+
+# File with translations
+i18n <- Translator$new(translation_csvs_path = "../data/")
+i18n$set_translation_language("en") # here you select the default translation to display
+
+ui_module <- function(id) {
+  ns <- NS(id)
+
+  tagList(
+    shiny.i18n::usei18n(i18n),
```

Comment:
Explicit namespace is not necessary, unless it is used for a good reason.

Reply:
Good point, that was just a copy paste from other examples :+1: